app/features/config/index.js

```diff
@@ -8,5 +8,20 @@ export default {
     /**
      * The domain of the Jitsi Meet deployment that will be used.
      */
-    defaultDomain: 'meet.jit.si'
+    defaultDomain: 'meet.jit.si',
+
+    /**
+     * The url of Privacy Policy Page.
+     */
+    privacyPolicyURL: 'https://jitsi.org/meet/privacy',
+
+    /**
+     * The url of Terms and Conditions Page.
+     */
+    termsAndConditionsURL: 'https://jitsi.org/meet/terms',
+
+    /**
+     * Email to send feedback.
+     */
+    feedbackEmail: 'support@jitsi.org'
```

Comment:
Let's make this "feedbackURL" and add the scheme. This way, if we ever want to change from a mailto URL to a HTTP URL it will be a single line change.

Reply:
Hmm. 👍